tests/x11/sshxterm.pm

```diff
@@ -15,16 +15,20 @@ sub run() {
     my $self = shift;
     mouse_hide(1);
     x11_start_program("xterm");
-    script_run("ssh -XC root\@localhost xterm");
-    type_string "yes\n";
-    wait_idle 6;
+
+    # check last incoming ssh traffic, wait until the connection is opened from localhost.
+    # "ESTAB", AND ::1:ssh, AND dport(destination-port) for control with exactitude, that a inc. port from ssh only from localhost is open.
+    # at this moment type password with serenity.
+    script_run("for i in {1..100}; do /usr/sbin/ss -at dport = :ssh | tail -n1 >/dev/$serialdev; sleep 2; done & ");
```

Comment:
that looks like a very clever way to do … what exactly? I am sorry but I don't get what this is supposed to do.